package.json

```diff
@@ -1,7 +1,35 @@
 {
-  "name": "azure-sdk-for-net",
```

Comment:
We need to add a copy of [this tsp](https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/Unbranded-TypeSpec.tsp) and a script which can generate that tsp using the azureplugin similar but much less complex to [Generate.ps1](https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/eng/scripts/Generate.ps1).

Reply:
Was planning to add test in next PR, think we have enough to get the initial setup.

Reply:
added